api/src/pcapi/routes/backoffice/chronicles/blueprint.py

```diff
@@ -128,7 +204,7 @@ def publish_chronicle(chronicle_id: int) -> utils.BackofficeResponse:
         chronicle=chronicle,
     )
     flash(f"La chronique {chronicle_id} à été publiée", "success")
```

Comment:
On avait laissé passer ça (et aussi dans `unpublish` en-dessous) : 
```suggestion
    flash(f"La chronique {chronicle_id} a été publiée", "success")
```